### PR TITLE
Installation doc update

### LINKED ISSUE #1983: Update the description and screenshots in docs/installation
Related #1850.

Now that Rufus is version 3.1 and Etcher is version 1.4.4, so their UI and how to operate them are a little different from them in our website. I think it's time to review and update the description and screenshots in docs/installation.

### PR DESCRIPTION
**Pull request by 5nai3r**
Fixes #1983

### Changes Summary

- Updated Rufus screenshot and steps 
- Added Gulp task for optimizing gif
- Updated Etcher screenshot

This pull request is ready for review.

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -110,20 +110,16 @@ To create an elementary OS install drive on macOS you'll need a USB flash drive
 
 <a href="http://www.etcher.io/" class="button suggested-action">Download Etcher</a>
```
Comment (ryonakano) [inline docs/installation.md]: I think this should be `https://etcher.io/`; the URL now supports HTTPS

**Comment by ryonakano** [comment]
@5nai3r I think you also should change the URL of UNetbootin to https://unetbootin.github.io/ (at line166)

File: _styles/docs.css [hunk 2]
```diff
@@ -103,6 +103,10 @@ ul {
     margin: 1.5em auto;
 }
 
+.docs img.inline {
```
Comment (lewisgoddard) [inline _styles/docs.css]: This should probably just be `img.inline` without the `.docs` scope.

File: docs/installation.md (hunk 1 above)
Comment (lewisgoddard) [reply docs/installation.md]: Agreed

**Comment by 5nai3r** [comment]
@lewisgoddard  @ryonakano  
thank you for your review 
i made what you requested on the commit 713cbf0 check it when you have time

**Comment by ryonakano** [comment]
@5nai3r Thank you for your fix, it works for me.

**Comment by 5nai3r** [comment]
@btkostner  Yes I'm running windows 10, and I have font isseus  since I switched to Windows 10, it's even worse in google chrome 
do you know any FIX for that font ? or it will be faster to take The screenshot from Win 7 in my laptop

**Comment by btkostner** [comment]
@5nai3r Sorry, I don't know a fix for that. It just looked really off.

**Pull request merged**